packages/vscode-extension/src/handlers.ts

```diff
@@ -1579,10 +1579,10 @@ export async function openHelpFeedbackLinkHandler(args: any[]): Promise<boolean>
   return env.openExternal(Uri.parse("https://aka.ms/teamsfx-treeview-helpnfeedback"));
 }
 export async function openWelcomeHandler(args?: any[]): Promise<Result<unknown, FxError>> {
-  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.QuickStart, getTriggerFromProperty(args));
```

Comment:
Don't forget to do telemetry data classification.

Reply:
I kept the telemetry event name unchanged. Can we also change the telemetry event name?